src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -552,9 +552,12 @@ void DlgPrefEQ::slotUpdateMainEQParameter(int value) {
                 EffectManifestParameter::ParameterType::Knob, index);
 
         if (pParameterSlot->isLoaded()) {
-            double dValue = value / 100.0;
-            pParameterSlot->setParameter(dValue);
+            // relative position for parameter
+            double paramValue = double(value - slider->minimum()) /
+                    double(slider->maximum() - slider->minimum());
```

Comment:
For some reason the division of int / int did not return a double but int.
Casting dividend & divisor to double fixes that :shrug:

Reply:
Yes, the compiler keep the type. Once one of the values is double, the result is also double. 
Just use `static_cast<double>() here to be more in line with the other code.

Reply:
Okay, done.